packages/salesforcedx-vscode-apex-replay-debugger/src/context/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { WorkspaceContext } from './workspaceContext';
```

Comment:
@xyc Let me know if this needs to be changed to comply with the webpack rules you mentioned earlier